test/unit_Polygon.jl

```diff
@@ -107,3 +107,23 @@ for N in [Float64, Float32] # TODO Rational{Int}
     @test ⊆(HPolygon{N}(), p1)
     @test ⊆(p1, BallInf(N[1., 0.], N(1.)))
 end
+
+# Test comparison of directions <= for rational entries
```

Comment:
We should put this whole block inside the above for loop after adding `Rational` to its arguments.